cmd/initialize/cmd.go

```diff
@@ -284,9 +238,82 @@ func run(cmd *cobra.Command, argv []string) {
 		}
 	}
 
+	// Verify version of `oc`
 	oc.Cmd.Run(cmd, argv)
 }
 
+func ocmLogin(cmd *cobra.Command, argv []string, reporter *rprtr.Object) error {
```

Comment:
This is repeated from account sub command. We can't abstract some of this to the ocm package?

Reply:
Good point, I'll fix that (I'm thinking inside `cmd/login/`, as I don't want to feed `cobra.Command` to the `ocm` package.

Reply:
Done. Moved to `cmd/login/cmd.go`